WordPress/Classes/Services/CommentService.m

```diff
@@ -22,6 +23,42 @@ @interface CommentService ()
 
 @implementation CommentService
 
++ (NSMutableSet *)syncingCommentsLocks
+{
+    static NSMutableSet *syncingCommentsLocks;
+    static dispatch_once_t onceToken;
+    dispatch_once(&onceToken, ^{
+        syncingCommentsLocks = [NSMutableSet set];
+    });
+    return syncingCommentsLocks;
+}
+
++ (BOOL)isSyncingCommentsForBlog:(Blog *)blog {
+    return [self isSyncingCommentsForBlogID:blog.objectID.URIRepresentation];
```

Comment:
I think storing the `NSManagedObjectID`s would work as well, and be faster than the `URIRepresentation`